package.json

```diff
@@ -74,6 +74,7 @@
     "express-session": "^1.15.1",
     "isomorphic-relay": "^0.7.4",
     "lodash": "^4.17.4",
+    "node-fetch": "^1.6.3",
```

Comment:
interesting! Looks like it's already in at that same spec level, you just made it specific as a dep now:

```
node-fetch@^1.0.1, node-fetch@^1.6.3:
  version "1.6.3"
  resolved "https://registry.yarnpkg.com/node-fetch/-/node-fetch-1.6.3.tgz#dc234edd6489982d58e8f0db4f695029abcd8c04"
  dependencies:
    encoding "^0.1.11"
    is-stream "^1.0.1"
```

It comes from `graphql-config-parser`